lang/de.json

```diff
@@ -375,7 +375,7 @@
     "cmishap-invalid-reference": "Ungültige Magisches Mißgeschickreferenz",
     "cmodule-description": "Wirft die Würfel für ein Mechatron-Modul und prüft auf Überhitzung.",
     "cmonster-description": "Zeigt ein Monster aus den Katalogen an oder erzeugt ein zufälliges Monster anhand der Tabellen im *Zonenkompendium 1: Das Versteck der Reptiloiden* falls keine Argumente übergeben wurden.",
-    "cmonster-more-descriptions": [
```

Comment:
Found another one!

Reply:
Yes, i also fixed that in my PR from earlier 😄